bin/tessel-2.js

```diff
@@ -323,7 +287,7 @@ parser.command('init')
   .callback(options => {
     log.level(options.loglevel);
 
-    callControllerWith('t2Init', options);
+    callControllerWith('createNewProject', options);
```

Comment:
This renaming choice was subjective, chosen to be more explicit about the operation performed